### ISSUE TITLE
Quest: Necklace Recovery, Take 3

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#61]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.4

Orgrimmar / The Drag / NPC: Dran Droffers / Quest: Necklace Recovery, Take 3
Link: https://www.wowhead.com/wotlk/quest=2341/necklace-recovery-take-3

Description: Despite the voice-over is perfect, the text is a description of actions. The narrator voice is more suitable in this case, instead of the male orc voice.

Completion: playback ok.

**Comment by carevass** [comment]
This should be fixed with new audio in AI_VoiceOverData_VanillaExtra_v1.5.
